### PR TITLE
Granite's fix for Red decals - aka my OCD can not handle seeing red as blue anymore in SDMM

### PR DESCRIPTION
**Pull request by Zergspower**
## About The Pull Request

I know WHY it was done and as such I didnt touch tha part of it, this just adds a new layer under Red called 'Real Red' so the option for a red decal/trim/etc are there to be used in other areas. The dark red at times is too 'syndie' for my tastes. I didnt add them to the paint-gun since those options in there are department based, but if someone thinks I should I could.

image
image

## How This Contributes To The Skyrat Roleplay Experience

Red

## Changelog

:cl:
qol: mappers can now use the actual color red in decals.
imageadd: adds real_red decals and logos to be able to be used for mapping
/:cl:

### CODE REVIEW

File: code/game/objects/effects/decals/turfdecal/markings.dm [hunk 1]
```diff
@@ -246,6 +246,19 @@
 /obj/effect/turf_decal/siding/red/end
 	icon_state = "siding_end"
 
+// Skyrat Edit: Start - Adds red variant that doesnt break 'bluesec' but allows mappers to use red elsewhere
```
Comment (Zonespace27) [inline code/game/objects/effects/decals/turfdecal/markings.dm]: modularize this

File: code/game/objects/effects/decals/turfdecal/tilecoloring.dm [hunk 2]
```diff
@@ -1257,3 +1257,108 @@
 
 /obj/effect/turf_decal/trimline/dark/filled/shrink_ccw
 	icon_state = "trimline_shrink_ccw"
+
+// Skyrat Edit: Start
```
Comment (Zonespace27) [inline code/game/objects/effects/decals/turfdecal/tilecoloring.dm]: modularize this

**Review by Zonespace27 — DISMISSED** [review]

File: code/game/objects/effects/decals/turfdecal/markings.dm (hunk 1 above)
Comment (Zergspower) [reply code/game/objects/effects/decals/turfdecal/markings.dm]: yeah was hoping to keep them in the same group

**Comment by ghost** [comment]
Honestly ---
Do we NEED to override the red color like this any more? Surely we've a better way to change the markings without changing every single map's markings. Could it not be a mapload check, area check, something besides "red is now blue"?

**Comment by Zergspower** [comment]
> Honestly --- Do we NEED to override the red color like this any more? Surely we've a better way to change the markings without changing every single map's markings. Could it not be a mapload check, area check, something besides "red is now blue"?

yeah it's pretty lazy how it was the first time - i'm all ears if you got an idea that doesnt involve breaking all the maps

**Comment by Zergspower** [comment]
@Ebin-Halcyon @Paxilmaniac i'm told you two need to sign off on this too

**Review by Ebin-Halcyon — APPROVED** [review]
They're red alright.

**Comment by Zergspower** [comment]
@Zonespace27

File: modular_skyrat/master_files/code/game/objects/effects/decals/turfdecals/sr_markings.dm [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// Adds red variant that doesnt break 'bluesec' but allows mappers to use red elsewhere
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/game/objects/effects/decals/turfdecals/sr_markings.dm]: Why is it called `sr_markings.dm` if it's not really markings, nor is it senior markings?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
The names of these files is just a bit weird to me. Like, `sr_` isn't a prefix we use for file names anywhere, and it's just a little odd.

File: modular_skyrat/master_files/code/game/objects/effects/decals/turfdecals/sr_markings.dm (hunk 3 above)
Comment (Zergspower) [reply modular_skyrat/master_files/code/game/objects/effects/decals/turfdecals/sr_markings.dm]: mostly did it to avoid duplicate naming

**Review by Paxilmaniac — APPROVED** [review]
bluesec and its consequences

**Review by GoldenAlpharex — APPROVED** [review]
Seems good, thanks!

**Pull request merged**